### ISSUE TITLE
Resource cutback

### ISSUE THREAD

**Issue by Mewchild** [HeavensGate/Eternal#344]
I also think it'd be nice if resources were a LOT more limited on the station, I believe this was talked about a long while ago when I'd only just become a mod, but this station is in the middle of nowhere, there should be some scarcity. Generally speaking, if there's no cargo techs or QM, nobody will notice. I think they should play a more vital role in things, and I'm not just saying that because I used to play nothing but CT...

Also, Mew, passive stuff just cut down on a few of the overly abundant things. Like... Um... Hmmm... All I can think of is medical stuff, but this should affect more than one department.

**Comment by Mewchild** [comment]
Curse you inability to assign multiple

**Comment by staticcharged** [comment]
Security could do with some minimizing?

**Comment by Finriv** [comment]
`Security could do with some minimizing?`

Make armory smaller, less lockers, The brig itself is rather large and stocked rather well (I think the whole brig should be remapped imo)

Armoury should be more like a Riot locker, less SMG's laser rifles and shiznit. They're here to protect the station not blow it up with military grade ordnance.

Cargo is fine
Can't say about medical.
Remove the drink dispensers from the bar, as long as they're there you have unlimited booze.
Science is fine, since most of their shit relies on making stuff their self (Read, slimes, mechs R&D)
Engineering, less lockers maybe? Not as much metal/glass and or vending machines perhaps.

Tl;dr, less vendging machines. more ways to make ghetto stuff, more physical presence (Maybe just less in vending machines and the ability to refill them, thanks Cookay)

**Comment by cookays** [comment]
The armory doesn't have SMGs. The only ballistic weapons in there are shotguns.

**Comment by Jacquar** [comment]
Glad Mew is bringing this up, as it has been a point of contention but
something I've called for loudly since the start. Scarcity promotes
roleplay, interaction, improvisation. Without needs, there's no feeling, no
purpose. Cargo should be the source for materials, in-situ gear at round
start should not be adequate for supply any department's full needs over a
90min+ round.
